commands/group_e2e_test.go

```diff
@@ -507,3 +507,44 @@ func createTestGroupTeam(s *MmctlE2ETestSuite) (*model.Team, *model.Group, func(
 
 	return team, group, cleanUpFn
 }
+
+func (s *MmctlE2ETestSuite) TestUserGroupRestoreCmd() {
+	s.SetupEnterpriseTestHelper().InitBasic()
+
+	// create group
+	id := model.NewId()
+	group, appErr := s.th.App.CreateGroup(&model.Group{
+		DisplayName: "dn_" + id,
+		Name:        model.NewString("name" + id),
+		Source:      model.GroupSourceCustom,
+		Description: "description_" + id,
+		RemoteId:    model.NewString(model.NewId()),
+	})
+	s.Require().Nil(appErr)
+	s.th.App.Srv().SetLicense(model.NewTestLicenseSKU(model.LicenseShortSkuProfessional, "ldap"))
+
+	defer func() {
+		_, err := s.th.App.DeleteGroup(group.Id)
+		s.Require().Nil(err)
+	}()
+
+	s.Run("Should allow group restore after deletion", func() {
```

Comment:
Do we allow for all client types (eg. user, admin and local user) to restore groups? I couldn't see a test regarding to authorization.

Reply:
https://github.com/mattermost/mmctl/pull/614/commits/5933073bb37fab78a838f709c4fd6226ec6633f7 @isacikgoz I've added a test, but it looks like in my naivety, I didn't create a role `PermissionRestoreCustomGroup` in https://github.com/mattermost/mattermost-server/pull/20369/files this PR.

I will create a PR to fix that, but do you think we should wait for that to merge this? Or can I just update this code when I create a new permission to restore groups.